releasenotes/notes/agent-xds-proxy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+issue: []
+releaseNotes:
+- |
+  **Updated** XDS connections from Envoy will be proxied through the Istio sidecar agent. Doing so allows
```

Comment:
question: how can I enable/disable this feature per pod

Reply:
Any env var can be configured by `proxyMetadata: Key: value`. This can be configured globally in mesh config `defaultConfig` or per pod with `proxy.istio.io/config` annotation.